### PR TITLE
updating toolset

### PR DESCRIPTION
**Pull request by frichtarik**

### CODE REVIEW

File: test/cdk_diff.bats [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bats
 
-load "cdk_diff.sh"
+load "../cdk_diff.sh"
```
Comment (polothy) [inline test/cdk_diff.bats:3]: Hrm, why is this needed now? Something change in `bats`?

Comment (frichtarik) [reply test/cdk_diff.bats:3]: honestly i'm not sure 
i tried to look at last action run for previous version but it's gone (more than a year) 

i did update bats library, there can be internal change in that, or ubuntu-latest image can behave a bit differently after a year ... i guess were never going to be sure

File: install_yq.sh [hunk 2]
```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
 
 download_yq() {
-  echo "⬇️ mikefarah/yq/releases/download/3.4.1/yq_linux_amd64 to yq"
-  wget --no-verbose -O yq https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_amd64
+  echo "⬇️ mikefarah/yq/releases/download/v4.40.5/yq_linux_amd64 to yq"
+  wget --no-verbose -O yq https://github.com/mikefarah/yq/releases/download/v4.40.5/yq_linux_amd64
```
Comment (frichtarik) [inline install_yq.sh]: finally realized why i wasn't able to download more recent version - they changed versioning from `3` to `v4` since 4.0.0

i'm going to fix tests now

**Comment by frichtarik** [comment]
i'm going to test it in some PR's first just to be sure

File: install_yq.sh (hunk 2 above)
Comment (polothy) [reply install_yq.sh]: This is actually pinned to an older version on purpose because the newer version was astronomically slower for some reason.  They may have fixed it, but be aware, the CLI interface changes frequently, so I doubt it'll work without further changes.

Personally, I wouldn't change this.

**Comment by frichtarik** [comment]
after testing in several repos - stabile and fast for both change and no change situations, no issue with downloading the yq binary 

if you're ok with it, i'll merge it 

my next plan is to create action - wrapper that would cover whole cdk-diff job, so that we don't have to update separate actions across the org

**Comment by polothy** [comment]
Please don't merge, I'd have to test against the forge.

**Comment by polothy** [comment]
Looks like `yq` is still very slow.  Granted, the forge has a silly number of stacks, but doing a diff prior to this change is around 38 seconds, doing a diff after this change, it's around 2.5 minutes.

**Comment by polothy** [comment]
If you could please revert the `yq` changes, I'd be happy to merge this.

**Comment by frichtarik** [comment]
@polothy thanks for spending time with this, yq upgrade is reverted, feel free to merge anytime

**Review by polothy — APPROVED** [review]
